src/VS.Web.CG.Mvc/Templates/Identity/Pages/Account/Manage/Account.Manage.DownloadPersonalData.cs.cshtml

```diff
@@ -0,0 +1,57 @@
+@inherits Microsoft.VisualStudio.Web.CodeGeneration.Templating.RazorTemplateBase
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+@{
+    var userClassName = "IdentityUser";
+    if (Model.IsGenerateCustomUser)
+    {
+@:using @(Model.DbContextNamespace);
+        userClassName = @Model.UserClass;
+    }
+}
+
+namespace @(Model.Namespace).Areas.Identity.Pages.Account.Manage
+{
+    public class DownloadPersonalDataModel : PageModel
+    {
+        private readonly UserManager<@(userClassName)> _userManager;
+        private readonly ILogger<DownloadPersonalDataModel> _logger;
+
+        public DownloadPersonalDataModel(
+            UserManager<@(userClassName)> userManager,
+            ILogger<DownloadPersonalDataModel> logger)
+        {
+            _userManager = userManager;
+            _logger = logger;
+        }
+
+        public async Task<IActionResult> OnPostAsync()
+        {
+            var user = await _userManager.GetUserAsync(User);
+            if (user == null)
+            {
+                return NotFound($"Unable to load user with ID '{_userManager.GetUserId(User)}'.");
+            }
+
+            _logger.LogInformation("User with ID '{UserId}' asked for their personal data.", _userManager.GetUserId(User));
+
+            // Only include personal data for download
+            var personalData = new Dictionary<string, string>();
+            personalData.Add("UserId", await _userManager.GetUserIdAsync(user));
+            personalData.Add("UserName", await _userManager.GetUserNameAsync(user));
+            personalData.Add("Email", await _userManager.GetEmailAsync(user));
+            personalData.Add("EmailConfirmed", (await _userManager.IsEmailConfirmedAsync(user)).ToString());
+            personalData.Add("PhoneNumber", await _userManager.GetPhoneNumberAsync(user));
+            personalData.Add("PhoneNumberConfirmed", (await _userManager.IsEmailConfirmedAsync(user)).ToString());
```

Comment:
This _userManager call doesn't match the dictionary key.

Reply:
@HaoK, I am assuming this isn't intentional and needs to fixed in https://github.com/aspnet/Identity/blob/dev/src/UI/Areas/Identity/Pages/Account/Manage/DownloadPersonalData.cshtml.cs#L45 as well.

Reply:
Yeah its not, I'll fix it in UI as well thanks